bin/acceptance-tests.sh

```diff
@@ -56,6 +56,7 @@ docker run \
     -e "BASE_URL=${BASE_URL:-}" \
     -e "BASE_POCKET_URL=${BASE_POCKET_URL:-}" \
     -e "PYTEST_PROCESSES=${PYTEST_PROCESSES:=4}" \
+    -e "BOUNCER_URL=${BOUNCER_URL:-}" \
```

Comment:
Do long as your app will default to the correct thing when the URL is set to an empty string this looks good

Reply:
Ah, nope - empty strings don't trigger the default param:

```~/Code/bedrock $ BOUNCER_URL="" ./manage.py shell
Using SITE_MODE of 'Mozorg'
>>> from django.conf import settings
>>> settings.BOUNCER_URL
''```

so will add it explicitly